app/redux/services/selectAccount.ts

```diff
@@ -8,8 +8,7 @@ import { getAllTxosForAccount } from './getAllTxosForAccount';
 
 export const selectAccount = async (accountId: string): Promise<void> => {
   try {
-    const { accountIds, accountMap } = await fullServiceApi.getAllAccounts();
-
+    const { accountIds, accountMap } = store.getState().accounts;
```

Comment:
don't fetch all accounts from the api every time we select an account. Rely on redux store for accounts.